packages/stryker/src/mutators/BooleanSubstitutionMutator.ts

```diff
@@ -0,0 +1,32 @@
+import { Mutator, IdentifiedNode } from 'stryker-api/mutant';
+import { Syntax } from 'esprima';
+
+export default class BooleanSubstitutionMutator implements Mutator {
+  name = 'BooleanSubstitution';
+  private type = Syntax.Literal;
+  private booleanValues: { [targetedOperator: string]: string } = {
+    'true': 'false',
+    'false': 'true'
+  };
+
+  applyMutations(node: IdentifiedNode, copy: <T>(obj: T, deep?: boolean) => T): IdentifiedNode[] {
+    let nodes: IdentifiedNode[] = [];
+    
+    // !a -> a
+    if (node.type === Syntax.UnaryExpression && node.operator === '!') {
+      let mutatedNode = copy(node.argument) as IdentifiedNode;
+      mutatedNode.nodeID = node.nodeID;
+      nodes.push(mutatedNode);
+    }
+
+    // true -> false or false -> true
+    if (node.type === this.type && this.booleanValues[node.raw]) {
+      let mutatedNode = copy(node);
+      mutatedNode.raw = this.booleanValues[node.raw];
+      mutatedNode.value = JSON.parse(mutatedNode.raw);
```

Comment:
I think setting the value should be enough. That way you can set it like this: `mutatedNode.value = !mutatedNode.value`